cmd/engine/engine.go

```diff
@@ -178,6 +178,30 @@ func doCmd(_ *cobra.Command, _ []string) (err error) {
 	assessmentService = service_assessment.NewService()
 	evidenceStoreService = service_evidenceStore.NewService()
 
+	assessmentService.RegisterAssessmentResultHook(func(result *assessment.AssessmentResult, err error) {
+
+		if err != nil {
+			log.Errorf("error was passed: %v", err)
+		}
+		_, err = orchestratorService.StoreAssessmentResult(context.Background(), &orchestrator.StoreAssessmentResultRequest{
+			Result: result})
+
+		if err != nil {
+			log.Errorf("error storing assessment result in orchestrator: %v", err)
+		}
+	})
+
+	// TODO(all): register assessment result hook
+	orchestratorService.RegisterAssessmentResultHook(func(result *assessment.AssessmentResult, err error) {
```

Comment:
Is this a real ToDo for here in Clouditor? I thought, we just define here the hook and the actual function is implemented and passed in the Clouditor Wrapper of MEDINA.

Reply:
That was an info for Immanuel, but I delete it before merging.